modules/extension/wms/src/test/java/org/geotools/data/wms/test/GetStyleOnlineTest.java

```diff
@@ -0,0 +1,136 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2004-2008, Open Source Geospatial Foundation (OSGeo)
+ *    (C) 2005, David Zwiers
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.data.wms.test;
+
+import org.geotools.data.ows.Specification;
+import org.geotools.data.wms.WMS1_1_1;
+import org.geotools.data.wms.WebMapServer;
+import org.geotools.data.wms.request.GetStylesRequest;
+import org.geotools.data.wms.response.GetStylesResponse;
+import org.geotools.styling.NamedLayer;
+import org.geotools.styling.SLDParser;
+import org.geotools.styling.Style;
+import org.geotools.styling.StyleFactory;
+import org.geotools.styling.StyleFactoryImpl;
+import org.geotools.styling.SLDTransformer;
+import org.geotools.styling.StyledLayerDescriptor;
+import org.geotools.styling.builder.NamedLayerBuilder;
+import org.geotools.styling.builder.StyledLayerDescriptorBuilder;
+
+import org.geotools.styling.builder.StyleBuilder;
+
+import org.geotools.test.OnlineTestCase;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import javax.mail.internet.ContentType;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.URL;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * 
+ * 
+ * @source $URL$
+ */
+public class GetStyleOnlineTest extends OnlineTestCase {
+
+    @Override
+    protected String getFixtureId() {
+        return "WMS";
+    }
+
+    @Test
+    public void testGetStyle() throws Exception {
```

Comment:
Since this is a single test you may be able to add it to the existing GeoServerOnlineTest?

The test itself look fine thank you.